src/components/EventsPage/EventsPage.module.scss

```diff
@@ -0,0 +1,44 @@
+@import "../../styles/_vars.scss";
+
+.pageTitle {
+  display: inline-block;
+  vertical-align: middle;
+}
+
+.newEventLink {
+  display: inline-block;
+  margin-left: 15px;
+}
+
+.eventsWrapper {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.eventCard {
+  margin: 0 20px 20px 0;
+  min-width: 400px;
+  transition: all $transitionDuration $transitionFunction;
+
+  img {
+    background: #e0e0e0;
+    height: 240px;
+    max-width: 100%;
+    width: 360px;
+  }
+  .eventName {
+    margin: 13px 0 10px 0;
```

Comment:
This hurts me on an emotional level 😞 Can we keep padding as even numbers (if you want to earn extra brownie points, aim to keep it a multiple of 4 if over 16, otherwise a multiple of 2) 😆